### PR TITLE
Implement new documents collection design

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Add storybook stories.
Export props from RawLink component.
Update translations (these are most of the changes hehe).

> Work in progress. Waiting for #635 to be merged.

## Why?

Updated designs by @zakkjellin on offer page.

_Referenced ticket(s): []_

## Demo

Mobile (hover state on second item)

Screenshot 2021-10-27 at 23 08 29

Desktop (hover state on first item)

Screenshot 2021-10-27 at 18 07 27

### CODE REVIEW

**Review by gustaveen — DISMISSED** [review]
giphy (4)

**Review by guilhermespopolin — DISMISSED** [review]
LGTM

**Comment by robinandeer** [comment]
I think I agree with @styrelius that we should avoid to introduce new patterns without discussing it with the group so I reverted the "dot notion" exports from the component.

Please re-review again @gustaveen @guilhermespopolin!

File: src/client/pages/OfferNew/Perils/DocumentCollection/CollapsingList.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    &:hover {
+      background-color: ${color(colorsV3.gray300)
+        .darken(0.03)
+        .toString()};
+    }
+
+    ${Container} &:not(:last-child) {
+      margin-right: 1rem;
+      margin-bottom: 1rem;
+    }
```
[the remark this answers is not in the document]
Comment (styrelius) [reply src/client/pages/OfferNew/Perils/DocumentCollection/CollapsingList.tsx:59]: Yeah, that's true 🧠

**Review by gustaveen — APPROVED** [review]
🙌

**Pull request merged**